mod/wet4/views/default/object/elements/summary.php

```diff
@@ -65,7 +62,14 @@
 	$tags = elgg_view('output/tags', array('tags' => $entity->tags));
 }
 
+
+if ( !$title_link && json_decode($entity->title) ){
```

Comment:
This line adds an additional header of the content title to the page. It appears right above the author details.